third_party/rust/curve25519_dalek/v3/crate/Cargo.toml.orig

```diff
@@ -50,7 +50,7 @@ serde = { version = "1.0", default-features = false, optional = true, features =
 # The original packed_simd package was orphaned, see
 # https://github.com/rust-lang/packed_simd/issues/303#issuecomment-701361161
 packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_bits"], optional = true }
-zeroize = { version = ">=1, <1.4", default-features = false }
+zeroize = { version = "1", default-features = false }
```

Comment:
Why do new need to patch the original Cargo.toml? Isn't it just for reference?

Reply:
no, it's used by `gnrt gen`